gpt4all-chat/main.cpp

```diff
@@ -21,6 +22,7 @@ int main(int argc, char *argv[])
 
     QGuiApplication app(argc, argv);
     QQmlApplicationEngine engine;
+    qmlRegisterSingletonInstance("logger", 1, 0, "Logger", Logger::globalInstance());
```

Comment:
This is only necessary if you intend to expose it to qml which I don't think you need...

Reply:
I had a plan, but then decided not to do any extra stuff around this. Will reremove this.